### DISCUSSION TITLE
How works the cache in Orval?

### DISCUSSION THREAD

**Discussion by nathanredblur** [orval-labs/orval#573] [Q&A]
When you go to the official site you can read this in the landing "Generate, valid, `cache` and mock in your frontend applications"
but I couldn't find how Orval use the cache and how can I control it.
- is caching per request?
- is caching per data value?
- is in memory or localStorage?
- how to define the cache rules for certain request (chache-first, no-cache, etc)?

Thank you

**Comment by la55u** [comment]
My guess is it's referring to the built in caching mechanism of the generated clients e.g. react-query
